storage/imx6ull-flash/flashdrv.c

```diff
@@ -447,16 +447,20 @@ static void flashdrv_setDevClock(int dev, int state)
 flashdrv_dma_t *flashdrv_dmanew(void)
 {
 	flashdrv_dma_t *dma = mmap(NULL, _PAGE_SIZE, PROT_READ | PROT_WRITE, MAP_UNCACHED, OID_NULL, 0);
-	dma->last = NULL;
-	dma->first = NULL;
+	if (dma != MAP_FAILED) {
+		dma->last = NULL;
+		dma->first = NULL;
+	}
```

Comment:
A little beyond that PR but the `setup_flash_info()` function does not check the value returned by the `flashdrv_dmanew()` function.

Reply:
Fixed (somewhat), thank you